pkg/pipelines/tasks/tasks.go

```diff
@@ -74,3 +73,20 @@ func createContainer(name string, image string, workingDir string, cmd []string,
 		Args:       args,
 	}
 }
+
+func createVolumeMounts() []corev1.VolumeMount {
+	return []corev1.VolumeMount{
+		corev1.VolumeMount{
+			Name:      "varlibcontainers",
+			MountPath: "/var/lib/containers",
+		},
+	}
+}
+
+func createVolumes(name string) []corev1.Volume {
+	return []corev1.Volume{
+		corev1.Volume{
+			Name: name,
+		},
+	}
```

Comment:
Can you check to see if these methods are needed/referenced other than unit tests?

Reply:
These functions are not referenced anywhere. I'll remove them